packages/playwright/src/worker/testInfo.ts

```diff
@@ -33,7 +33,7 @@ export interface TestStepInternal {
   complete(result: { error?: Error, attachments?: Attachment[] }): void;
   stepId: string;
   title: string;
-  category: 'hook' | 'fixture' | 'test.step' | 'expect' | string;
+  category: 'hook' | 'fixture' | 'test.step' | 'expect' | 'pw:api' | string;
```

Comment:
I think it's a layering violation to list `pw:api` here in the test runner. Instead, we can explicitly blocklist all other categories in line 255. WDYT?

Reply:
Replaced with `canNestByTime` as discussed off the thread.